[#74334] 17.4 Meetings update  https://community.openproject.org/wp/7… (#22989)
opf/openproject: docs/user-guide/meetings/recurring-meetings/README.md

Author: Hi @MayaBerd I put a [!Note] to emphasize that meeting outcomes will not be duplicated and will only be copied when an agenda item is moved. Is that okay?

![Move an agenda item to next meeting in OpenProject recurring meetings](openproject_userguide_meetings_recurring_move_agenda_item.png)

> Clicking this option will display a confirmation dialog.  You will also be required to select the destination of the item from the drop-down menu: either to a desired section of the agenda or to the backlog. 

Reviewer: ```suggestion
Clicking this option will display a confirmation dialog. From there, choose where to move the item: a section of the agenda or the backlog.
```

This can be useful, for example, if an item was discussed but needs a follow-up in the next meeting, while the current meeting minutes should remain complete.

> To do duplicate an agenda item into the next meeting, open the three-dot **More (⋯)** menu of an agenda item and select Duplicate → Duplicate in next meeting. 

Reviewer: ```suggestion
To duplicate an agenda item into the next meeting, open the **More (...)** menu of an agenda item and select Duplicate → Duplicate in next meeting. 
```

![Duplicate an agenda item to next meeting in OpenProject recurring meetings](openproject_userguide_meetings_recurring_duplicate_agenda_item2.png)

> Clicking this option will display a confirmation dialog. You will also be required to select the destination of the item from the drop-down menu: either to a desired section of the agenda or to the backlog. 

Reviewer: ```suggestion
Clicking this option will display a confirmation dialog. You will then need to select a destination from the dropdown menu: either a section of the agenda or the backlog.
```
